nixos/modules/services/misc/airsonic.nix

```diff
@@ -116,6 +124,9 @@ in {
           -Dserver.port=${toString cfg.port} \
           -Dairsonic.contextPath=${cfg.contextPath} \
           -Djava.awt.headless=true \
+          ${if (cfg.virtualHost != null)
+            then "-Dserver.use-forward-headers=true"
+            else ""} \
```

Comment:
You can use `optionalString` instead of `if ... then ... else ""` here.

Reply:
Done